deltachat-ios/Chat/Reactions/ReactionsOverviewTableViewCell.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ReactionsOverviewTableViewCell.swift
+//  deltachat-ios
+//
+//  Created by Nathan Mattes on 16.01.24.
+//  Copyright © 2024 merlinux GmbH. All rights reserved.
+//
+
```

Comment:
in  general, we do not use such headers: git is enough for contributors, copyright and file name repetition not needed and anyways easily outdated

```suggestion
```